python/az/aro/azext_aro/azext_metadata.json

```diff
@@ -1,4 +1,4 @@
 {
     "azext.minCliCoreVersion": "2.0.67",
-    "version": "1.0.0"
+    "version": "2.0.0"
```

Comment:
I don't think 2.0.0 is warranted - let's go to 1.0.1

Reply:
done